go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/keptn/keptn/helm-service
+module github.com/keptn-contrib/helm-service
```

Comment:
The `go.sum` file is missing

Reply:
the `go.sum` file is already in the repo.